hazelcast/src/test/java/com/hazelcast/map/EntryProcessorTest.java

```diff
@@ -265,9 +263,10 @@ public void testEntryProcessorDeleteWithPredicate() {
 
     @Test
     public void testEntryProcessorWithKey() {
+        Config cfg = getConfig();
```

Comment:
`getConfig` method will be overridden in enterprise part and it will provide a different `Config` object.